### PR TITLE
Fix imports to new package

### PR DESCRIPTION
**Pull request by mfcastellani**

### CODE REVIEW

**Review by marioidival — APPROVED** [review]
SGTM 👀  ✅

File: go.mod [hunk 1]
```diff
@@ -3,14 +3,10 @@ module github.com/hermeznetwork/hermez-go-sdk
...
 
 require (
-	github.com/btcsuite/btcutil v0.0.0-20190425235716-9e5f4b9a998d // indirect
 	github.com/dghubble/sling v1.3.0
-	github.com/ethereum/go-ethereum v1.10.2
-	github.com/gobuffalo/logger v1.0.3 // indirect
-	github.com/hermeznetwork/hermez-node v1.4.0-rc2.0.20210630142508-555053e317e2
-	github.com/huin/goupnp v1.0.1-0.20210310174557-0ca763054c88 // indirect
+	github.com/ethereum/go-ethereum v1.10.4
+	github.com/hermeznetwork/hermez-node v1.5.0-rc2.0.20210713144540-6a0bb24d49e4
```
Comment (tclemos) [inline go.mod:8]: Is this version the correct one?

**Review by tclemos — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (jeffprestes) [reply go.mod:8]: There is a new release: v1.5.0 
I would recommend to update it.

**Review by jeffprestes — CHANGES_REQUESTED** [review]

**Comment by mfcastellani** [comment]
@jeffprestes version v1.5.0 does not include the latest atomic changes

```
❯ go get github.com/hermeznetwork/hermez-node@v1.5.0                                  
go: downloading github.com/hermeznetwork/hermez-node v1.5.0
go get: upgraded github.com/hermeznetwork/hermez-node v1.5.0-rc2.0.20210713144540-6a0bb24d49e4 => v1.5.0

❯ go mod tidy  

# at examples folder, in atomic example
❯ go run .   
# github.com/hermeznetwork/hermez-go-sdk/transaction
../../transaction/atomic.go:66:10: localTx.TokenSymbol undefined (type "github.com/hermeznetwork/hermez-node/common".PoolL2Tx has no field or method TokenSymbol)
../../transaction/atomic.go:126:13: fullTxs[i].RqOffset undefined (type "github.com/hermeznetwork/hermez-node/common".PoolL2Tx has no field or method RqOffset)
../../transaction/atomic.go:132:35: undefined: "github.com/hermeznetwork/hermez-node/common".AtomicGroup
../../transaction/atomic.go:147:17: undefined: "github.com/hermeznetwork/hermez-node/common".AtomicGroup
../../transaction/http.go:69:66: undefined: "github.com/hermeznetwork/hermez-node/common".AtomicGroup
```

**Pull request merged**